app/frontend/stylesheets/pages/partials/_footer.scss

```diff
@@ -0,0 +1,110 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.top-wave {
+  background-image: url("../images/wave-green.png");
+  height: 25vh;
+  background-color: white;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: bottom;
+  position: relative;
+}
+
+
+footer {
+  background-color: #85B6AA;
+  height: 60vh;
+  width: 100%;
+  position: absolute;
+  overflow: hidden;
+
+}
+
+.footer-container {  display: grid;
+  grid-template-columns: 1fr 1.5fr;
+  grid-template-rows: 100px 1fr 1fr 120px;
+  gap: 5px 2px;
+  grid-auto-flow: row;
+  grid-template-areas:
+    "content-logo content-links"
+    "content-logo content-socials"
+    "content-logo content-socials"
+    "content-logo content-copyright";
+}
+
+.content-logo {
+  grid-area: content-logo;
+  z-index: 10;
+}
+
+.content-links {
+  grid-area: content-links;
+}
+
+.content-socials {
+  grid-area: content-socials;
+}
+
+.content-copyright {
+  grid-area: content-copyright;
+  align-self: end;
+}
+
+.black-logo {
+  position: absolute;
+  left: 160px;
+  bottom: 130px;
+}
```

Comment:
Im struggling to wrap my head around the positions in css, ive still given each of the images the position absolute because if not they wont stay on the position where i want them to be. Perhaps you can help me with this.